action/recipes/stitching_test.go

```diff
@@ -141,6 +141,12 @@ type makeFile struct {
 }
 
 func (base makeFile) MakeMe() error {
+	log.Printf("Path:       %s", base.Path)
+	log.Printf("Content:    %s", base.Content)
+	log.Printf("SourcePath: %s", base.SourcePath)
+	pwd, _ := os.Getwd()
```

Comment:
I'm surprised the linter didn't complain here about the ignored error. It's totally fine here to drop it but still odd that you didn't need `nolint:errcheck`

Reply:
True, not sure why that is, so I added `nolint:errcheck` to future-proof this.